airflow/providers/docker/operators/docker_swarm.py

```diff
@@ -121,12 +153,16 @@ def _run_service(self) -> None:
                     env=self.environment,
                     user=self.user,
                     tty=self.tty,
+                    configs=self.configs,
+                    secrets=self.secrets,
                 ),
                 restart_policy=types.RestartPolicy(condition='none'),
                 resources=types.Resources(mem_limit=self.mem_limit),
+                networks=self.networks,
             ),
             name=f'airflow-{get_random_string()}',
             labels={'name': f'airflow__{self.dag_id}__{self.task_id}'},
+            mode=types.ServiceMode(mode=self.mode, replicas=self.replicas),
```

Comment:
Any reason why this isn’t set by passing in a `ServiceMode` instance but two separate flags?

Reply:
Thanks @uranusjr for your review.
I did not get your remark. Can you please elaborate ?
In all cases, I followed the signature of the function `create_service` as defined on the docker sdk for python : 
https://docker-py.readthedocs.io/en/stable/api.html#module-docker.api.service

Reply:
I meant why not just

```python
def __init__(
    self,
    ...,
    mode: ServiceMode = types.ServiceMode(mode="replicated", replicas=1),
    ...,
) -> None:
```

instead.

Reply:
Yes possible.
I just wanted to avoid as much as possible exposing to airflow users docker objects.

Reply:
Yeah that’s a good policy, and I would have recommended the same normally. But we are already exposing `ConfigReference` and `SecretReference` and tie-coupling to Docker.py, so it’s not very meaningful to not expose it anymore. If we want to hide it, we need to wrap *all* the configurations.

Reply:
👍 You got a point. Will modify it then

Reply:
@uranusjr Changes done as requested. Can you please review / approve ?